src/js/TuningSliders.js

```diff
@@ -368,24 +391,27 @@ TuningSliders.calculateNewPids = function() {
     $('output[name="sliderDMinRatio-number"]').val(this.sliderDMinRatio);
     $('output[name="sliderFFGain-number"]').val(this.sliderFFGain);
 
-    FC.TUNING_SLIDERS.slider_pids_mode = parseInt($('#sliderPidsModeSelect').val());
-    FC.TUNING_SLIDERS.slider_master_multiplier = TuningSliders.sliderMasterMultiplier * 100;
-    FC.TUNING_SLIDERS.slider_roll_pitch_ratio = TuningSliders.sliderRollPitchRatio * 100;
-    FC.TUNING_SLIDERS.slider_i_gain = TuningSliders.sliderIGain * 100;
-    FC.TUNING_SLIDERS.slider_pd_ratio = TuningSliders.sliderPDRatio * 100;
-    FC.TUNING_SLIDERS.slider_pd_gain = TuningSliders.sliderPDGain * 100;
-    FC.TUNING_SLIDERS.slider_dmin_ratio = TuningSliders.sliderDMinRatio * 100;
-    FC.TUNING_SLIDERS.slider_ff_gain = TuningSliders.sliderFFGain * 100;
-
-    FC.TUNING_SLIDERS.slider_dterm_filter = TuningSliders.sliderDTermFilter ? 1 : 0;
-    FC.TUNING_SLIDERS.slider_dterm_filter_multiplier = TuningSliders.sliderDTermFilterMultiplier * 100;
-
-    FC.TUNING_SLIDERS.slider_gyro_filter = TuningSliders.sliderGyroFilter ? 1 : 0;
-    FC.TUNING_SLIDERS.slider_gyro_filter_multiplier = TuningSliders.sliderGyroFilterMultiplier * 100;
-
-    MSP.promise(MSPCodes.MSP_SET_TUNING_SLIDERS, mspHelper.crunch(MSPCodes.MSP_SET_TUNING_SLIDERS));
-    MSP.send_message(MSPCodes.MSP_PID);
-    MSP.send_message(MSPCodes.MSP_PID_ADVANCED);
+    if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
+        FC.TUNING_SLIDERS.slider_pids_mode = parseInt($('#sliderPidsModeSelect').val());
+        FC.TUNING_SLIDERS.slider_master_multiplier = TuningSliders.sliderMasterMultiplier * 100;
+        FC.TUNING_SLIDERS.slider_roll_pitch_ratio = TuningSliders.sliderRollPitchRatio * 100;
+        FC.TUNING_SLIDERS.slider_i_gain = TuningSliders.sliderIGain * 100;
+        FC.TUNING_SLIDERS.slider_pd_ratio = TuningSliders.sliderPDRatio * 100;
+        FC.TUNING_SLIDERS.slider_pd_gain = TuningSliders.sliderPDGain * 100;
+        FC.TUNING_SLIDERS.slider_dmin_ratio = TuningSliders.sliderDMinRatio * 100;
+        FC.TUNING_SLIDERS.slider_ff_gain = TuningSliders.sliderFFGain * 100;
+
+        FC.TUNING_SLIDERS.slider_dterm_filter = TuningSliders.sliderDTermFilter ? 1 : 0;
+        FC.TUNING_SLIDERS.slider_dterm_filter_multiplier = TuningSliders.sliderDTermFilterMultiplier * 100;
+
+        FC.TUNING_SLIDERS.slider_gyro_filter = TuningSliders.sliderGyroFilter ? 1 : 0;
+        FC.TUNING_SLIDERS.slider_gyro_filter_multiplier = TuningSliders.sliderGyroFilterMultiplier * 100;
+
+        Promise.resolve(true)
+        .then(() => { return MSP.promise(MSPCodes.MSP_SET_TUNING_SLIDERS, mspHelper.crunch(MSPCodes.MSP_SET_TUNING_SLIDERS)); })
+        .then(() => { return MSP.send_message(MSPCodes.MSP_SET_PID); })
+        .then(() => { return MSP.send_message(MSPCodes.MSP_SET_PID_ADVANCED); });
```

Comment:
These were changed from ```MSP_PID```/```MSP_PID_ADVANCED``` to ```MSP_SET_PID``` /```MSP_SET_PID_ADVANCED```. I think this is what breaks it.

Reply:
@klutvott123 Yes! Thanks. Will make a patch later this day.

Reply:
Well spotted @klutvott123!